### ISSUE TITLE
ReferenceError in version 2.3

### ISSUE THREAD

**Issue by svedaei** [JoseGoncalves/vue-keycloak#2]
I've got an error as I wanted to run unit-test. 

Error:
ReferenceError: require is not defined in ES module scope, you can use import instead
This file is being treated as an ES module because it has a '.js' file extension and '*******/frontend/node_modules/@josempgon/vue-keycloak/package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.

**Comment by JoseGoncalves** [comment]
Hi @svedaei. Can you give a little more context on what you are trying to do? In what way are you trying to use this plugin that triggers that error?

**Comment by MichaelBrandt4994** [comment]
Hi @JoseGoncalves 
This is the full error we get
```
ReferenceError: require is not defined in ES module scope, you can use import instead
This file is being treated as an ES module because it has a '.js' file extension and '..../node_modules/@josempgon/vue-keycloak/package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.
 ❯ node_modules/@josempgon/vue-keycloak/dist/index.js:3:18
 ❯ ....ts:1:31
      1| import { useKeycloak } from "@josempgon/vue-keycloak"
       |                               ^
      2| import type { KeycloakConfig, KeycloakInitOptions } from "keycloak-js"
```

Actually, this only happens when we executing `vitest`. A build is running without problems.

**Comment by JoseGoncalves** [comment]
Never used `vitest` but, looking at the output you have supplied, I can only assume that something should not be correctly setup in your tests. Can you provide some small code/test environment in order I can reproduce that issue?

**Comment by JoseGoncalves** [comment]
Hi @MichaelBrandt4994 I think I should have fixed your issue in [v2.3.1](https://github.com/JoseGoncalves/vue-keycloak/releases/tag/v2.3.1). That version should not require any extra config in vitest. Please check it.

**Comment by MichaelBrandt4994** [comment]
Yes it is working without an extra config, Thanks! But we receive now some linting errors :smile: 

`vue-tsc --noEmit -p tsconfig.vitest.json --composite false`
```
Could not find a declaration file for module '@josempgon/vue-keycloak'. '.../node_modules/@josempgon/vue-keycloak/dist/index.esm.js' implicitly has an 'any' type.
  There are types at '.../node_modules/@josempgon/vue-keycloak/dist/types/index.d.ts', but this result could not be resolved when respecting package.json "exports". The '@josempgon/vue-keycloak' library may need to update its package.json or typings.

3 import { vueKeycloak } from "@josempgon/vue-keycloak"
```

But i will investigate this.

**Comment by JoseGoncalves** [comment]
@MichaelBrandt4994 If you find a solution for that issue please tell me... I could also check that on my side, if you can provide me a sample project that emits that linting error.

**Comment by JoseGoncalves** [comment]
Hi @MichaelBrandt4994 I was able to reproduce your latest issue and fixed it in release [v2.3.3](https://github.com/JoseGoncalves/vue-keycloak/releases/tag/v2.3.3). Please check if that also works for you.

**Comment by MichaelBrandt4994** [comment]
Yes it works :tada: 
Thanks!
